tests/e2e/actions.go

```diff
@@ -268,72 +395,231 @@ type SubmitConsumerAdditionProposalAction struct {
 	AllowInactiveVals   bool
 }
 
+func (tr Chain) UpdateConsumer(providerChain ChainID, validator ValidatorID, update types.MsgUpdateConsumer) {
+
+	fmt.Println("Update proposal for consumer_id=", update.ConsumerId)
```

Comment:
Do we want to print those always or only in case of `if verbose`? From what I see, we usually do printing/logging only when `verbose` is set.